ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd

```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2012, Red Hat, Inc., and individual contributors
```

Comment:
My watch says its 2020.

Reply:
Previously I thought I would change it to 2020 but I was not sure how the copyright think works.You can check now, I have made the changes. :)